ethereum/migrations/prod-receiver/3_deploy_pyth.js

```diff
@@ -1,5 +1,9 @@
-require('dotenv').config({ path: "../.env" });
-const bs58 = require("bs58");
+const dotenv = require("dotenv")
+dotenv.config({ path: "../../.env" });
+
+if (process.env.CLUSTER !== undefined) {
+  dotenv.config({ path: `../../.env.cluster.${process.env.CLUSTER}`});
+}
```

Comment:
extract this code out to a utility function like `load_dotenv()`

also, fwiw I think using a different config file solution is probably a good idea here. If we used something like HOCON that supports storing dictionaries in the config, then we wouldn't have to have separate config files for each network. `.env` is a rather inflexible standard, even though it is the "standard" for javascript.

Reply:
I looked up HOCON and although it’s interesting, it doesn’t have good adoption. Personally I like it’s features (mostly extending another config) but I don’t like it’s flexibility and it’s frustrating.
It had two parser pkgs in JS. The bigger one does not support include (which we need) and the other one was fairly new and not used much.